packages/integrations/cloudflare/src/runtime.ts

```diff
@@ -0,0 +1,7 @@
+
+export function getRuntime(request: Request): any {
```

Comment:
Is there a better return type than `any` we can give here? Does this help? https://github.com/cloudflare/workers-types

Reply:
yes that helps, unfortunately i did not manage to re-export the types, so i had to redeclare them. Also there are two possible types `WorkerRuntime` and `PagesRuntime` with slightly different properties.